README.md

```diff
@@ -1,5 +1,10 @@
 # The `ops` library
 
+![Charmcraft pack](https://github.com/canonical/operator/actions/workflows/charmcraft-pack.yaml/badge.svg)
+![Compatibility: DB charms](https://github.com/canonical/operator/actions/workflows/db-charm-tests.yaml/badge.svg)
+![Compatibility: Basic charms](https://github.com/canonical/operator/actions/workflows/hello-charm-tests.yaml/badge.svg)
+![Compatibility: Observability charms](https://github.com/canonical/operator/actions/workflows/observability-charm-tests.yaml/badge.svg)
+![CI Status](https://github.com/canonical/operator/actions/workflows/framework-tests.yaml/badge.svg)
```

Comment:
Hmm, I'm wondering if we really need/want all of these. Maybe just the Charmcraft pack and CI Status ones?

Reply:
I don't think they are super useful in the README itself. I'm not sure what the aim is with the releases page. @jnsgruk what workflows would you like to see showing up there?

Reply:
I think probably just the `ops test` workflow and the `publish` workflow?

Reply:
Thanks, changed!